### ISSUE TITLE
React: Masthead - With tab/shift tab key, focus is not going to the profile elements.

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#4200]
## Detailed description

Issue: 
With tab/shift tab key, focus is not going to the "My IBM, Log in" elements.

Expected :
With tab/shift tab key, Focus should go to each and every interactive element and it should be operable thru keyboard.
